static-catalog/README.md

```diff
@@ -1,163 +1,215 @@
-So both forms of STAC can be crawled in the same way.
+## Static Catalog Definitions
 
-The STAC JSON specifications for the JSON representation of elements returned by a  STAC.
-It also defines the JSON files that will be found in a Static Catalog.
+There are two required element types of a STAC Static Catalog: `Catalog` and `Item`.
+An `item` contains one or more `assets` which are artifacts defining some kind of media or product, such as a geotiff.
+A `Catalog` points to `Item`s, or to other `Catalog`s.
+The top-most parent `Catalog` is called the "root" catalog.
+The root catalog generally defines information about the catalog as a whole, such as name, description, licensing, contact information and so forth.
+Catalogs below the root generally have less information and serve to create a directory structure for categorizing and grouping `item` data.
+The contents of a `catalog` are flexible and STAC makes no assumptions for where or how catalog metadata is defined within a `catalog`.
+For example, a non-root catalog could redefine or add different licensing or copyright terms.
 
-## Static Catalog Definitions
+A simple Catalog structure might look like this:
+```
+catalog (root)
+  - catalog
+    - catalog
+      - item
+        - asset
+      - item
+        - asset
+    - item
+      - asset
+      - asset
+```
+This example might be considered a somewhat "typical" structure.
+However, `catalog`s and `item`s can describe a number of different relationships.
+The following shows various relationships between `catalog`s and `item`s:
 
-Static Catalogs define a network of linked assets for the purpose of automated
-crawling. The top level element of a Static Catalog is one or more linked `Catalog`s.
+- `Catalog` -> `Item` (this is a common structure for a `catalog` to list links to `items`)
+- `Catalog` -> `Catalog` (this is a common tree structure to group sets of items. Each `catalog` in this relationship may also include `item` links as well as `catalog` links)
+- `Item` -> `Catalog` (example: an item may point to a catalog to describe a set of derived assets, where it may be desirable to have the origin asset as a "parent", such as NDVI generated from RGB/IR)
+- `Item` -> `Item` (example: this relationship may be used to describe a 1-1 parent-child relationship, such as a single derived item from one parent item)
 
-There are two requred element types of a STAC Static Catalog: `Catalog`s and `Item`s. 
+The core `Item`s listed in a static catalog are the exact same form as those returned by a Catalog API.
+Ideally, STAC enables both a static and dynamic API to be crawled in the same way.
 
-The core STAC `Item` is defined in the [JSON Spec](../json-spec/), and is of the same form as those
-returned from a Catalog API, but the following will explain it in context, as well as the other element types.
+An `Item` in STAC is defined as a WFS 3.0 `Item`.
```

Comment:
Maybe, 'An `item` in STAC is compatible with the WFS 3.0 definition of an `item`'.